addon/utils.js

```diff
@@ -1,6 +1,9 @@
 import _ from 'lodash'
 import Ember from 'ember'
-const {A} = Ember
+const {A, get} = Ember
+import {getModelPath} from 'bunsen-core/utils'
+
+const assign = Ember.assign || Object.assign || Ember.merge
```

Comment:
Yes and `Object.assign` isn't available in any version of IE according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign. I believe it also isn't available in the version of Firefox Travis uses.